sz/Traffic/src-gen/traffic/ReLogoLink.java

```diff
@@ -296,5 +296,68 @@ public boolean isUserLinkQ(Object o){
 		return userLink(oneEnd.getWho(), otherEnd.getWho());
 	}
 
+	/**
+	 * Returns the value of the global variable intersectionType.
+	 *
+	 * @return the value of the global variable intersectionType
+	 */
+	@ReLogoBuilderGeneratedFor("global: intersectionType")
```

Comment:
Do you think that we need to include these files from `src-gen` in our commits?

Reply:
If Repast is able to regenerate these files after pulling, then we can skip them, but I'm not sure if that's the case.

Reply:
Nope, we need to keep them 🙁